dataikuapi/dss/ml.py

```diff
@@ -801,9 +816,12 @@ def _pretty_repr(self):
     def set_values(self, values):
         """
         Enables the search over listed values (categories).
-        :param values: values to enable, all other values will be disabled
+
+        :param values: list of values to enable, all other values will be disabled
         :type values: list of str
         """
+        assert isinstance(values, list) and not isinstance(values, string_types), \
```

Comment:
Isn't `isinstance(values, list)` enough ?

Reply:
Hum yes, indeed... I once considered testing more largely against iterables and thus explicitly exclude strings but after all it's so easy to make any iterable a `list` that I decided to stick to `list`.